src/foam/graph/GraphBuilder.js

```diff
@@ -39,7 +39,11 @@ foam.CLASS({
 
       // Iterate over rootObject's children
       var parent = this.data[rootObject.id];
-      return rootObject[relationshipKey].dao
+      var relationshipDAO = rootObject[relationshipKey].dao 
```

Comment:
Could be rootObject[relationshipKey].dao || ootObject[relationshipKey];

Reply:
Changed.